utils/helpers.js

```diff
@@ -0,0 +1,21 @@
+export function setHeaders(
+ res, headers = {}
```

Comment:
The first argument is `headers`, the second one is overriding the first one, give better names please.